packages/components/src/interfaces/complex/LogoCloud.ts

```diff
@@ -0,0 +1,39 @@
+import type { Static } from "@sinclair/typebox"
+import { Type } from "@sinclair/typebox"
+
+export const LogoCloudSchema = Type.Object(
+  {
+    type: Type.Literal("logocloud", { default: "logocloud" }),
```

Comment:
perhaps we can make `logocloud` a const to be passed around instead (esp. to studio) instead of using string literal. this will ensure single source of truth and help with debugging/development down the road 

similar to `DYNAMIC_DATA_BANNER_TYPE`

Reply:
sure